documentation/SupportingDocuments/CAMARA-Release-Workflow-and-Metadata-Proposal.md

```diff
@@ -0,0 +1,272 @@
+
+- **Limited CI Enforcement on `main`:**  
+  While MegaLinter is integrated for YAML and Gherkin, there is no structured validation for metadata files or enforcement of CAMARA-specific release guidelines.
+
+- **Unclear Versioning and Branching Strategy:**  
+  Metadata and version fields are frequently changed manually in `main` prior to release, only to be reset afterward. Changes made directly in release PRs are hard to trace and review.
+
+- **Minimal Automation for Changelogs and Checklists:**  
+  CHANGELOG generation is mostly manual. The API Readiness Checklist is not fully automated and depends on manual cross-checks.
+
+## Proposed Objectives and Guiding Principles
+
+We are proposing the following objectives to modernize and automate the CAMARA release process:
+
+- ✅ Clearly separate development intent (planning) from released state (actual).
+- ✅ Keep the `main` branch in a consistent "work in progress" (WIP) state with version fields as `"wip"`.
+- ✅ Use dedicated **release branches per (pre-)release** (e.g. for alpha, rc, release).
+- ✅ Introduce structured **metadata files (in YAML)** to support automation and CI.
+- ✅ Keep CAMARA **release numbering (`rX.Y`)** distinct from API **semantic versioning**.
+- ✅ Enforce CI gating on PRs to `main`.
+- ✅ Restrict release branch changes to authorized release managers.
+- ✅ Automate creation of release branches and generation of release metadata.
+- ✅ Use **Release preparation PRs** into release branches for structured approvals.
+- ✅ Handle specification or implementation feedback via PRs into `main`, and re-update the release branch as necessary.
+
+## Terminology
+
+| Term                   | Description |
+|------------------------|-------------|
+| `meta_release`         | Meta-release timeline (e.g., `Fall26`) |
+| `release_number`       | CAMARA release identifier within (e.g., `r4.1`). Distinct from API SemVer. |
+| `release_status`       | Status of the (planned) (pre-)release: `alpha`, `rc`, or `release`. |
+| `api_status`           | Per-API maturity used for gating validations. |
+| `main_contacts`        | GitHub handles of code owners or maintainers (per API in `release-plan.yaml`). |
+| `main` branch          | Development branch. All content is work-in-progress (`version: wip`). |
+| Release branch         | Dedicated release preparation branch per (pre-)release (e.g., `release/r4.1`). |
+| Release preparation PR | Pull request against a release branch to finalize or tweak the release content. |
+
+## Metadata File Format
+
+All metadata files are YAML-based and versioned in the repository. They serve as authoritative configuration & release inputs.
+
+### 1. `release-plan.yaml` (on `main`)
+
+Planning metadata owned by codeowners, manually updated and CI-validated.
+
+```yaml
+meta_release: Fall26 # default as long not eligible or planned for a meta-release: Other
```

Comment:
Is this always showing the latest meta-release information, overwriting previous info? Maybe this could be an array with a new item per meta-release, preserving the history, e.g.

```
- meta_release: Fall26
  ....
- meta_release: Spring26
  ...
```